trl/trainer/utils.py

```diff
@@ -19,11 +19,17 @@
 
 import numpy as np
 import torch
+from accelerate import Accelerator
```

Comment:
```suggestion
from accelerate import PartialState
```

Reply:
One should use `PartialState` whenever possible and not `Accelerator` in order to avoid initializing things that are not needed